cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,39 @@
+class TimeLoggingPage{
+    getUrl(){
+        return cy.url()
+    }
+
+    getPageTitle(){
+        return cy.get('.page__title')
+    }
+
+    getCalendar(){
+        return cy.get('.calendar')
+    }
+
+    getCalendarToday(){
+        return cy.get('.calendar--today')
+    }
+
+    getLogHoursForm(){
+        return cy.get('.tile.form')
+    }
+
+    getUserInfoTitle(){
+        return  cy.get('.user-info__title')
+    }
+
+    findMainNavTabs(){
```

Comment:
I think this could be `getMainNavTabs`, since what you actually do is get these tabs, even if the method is called "find" :)